### PR TITLE
Repo language cleanup

### PR DESCRIPTION
**Pull request by josh-chamberlain**
Makes the language on our "front door" representative of our current strategy. Solves some of #197, but leaves much to do as far as actually moving things around and reorganizing to match the vision.

Also removes a good deal of infrastructure and words left over from when we thought 
- we were going to store all our data in dolthub
- we were going to maintain our data sources in dolthub
- we were going to be doing ETL into a mega-database

### CODE REVIEW

File: setup_gui/README.md
[the remark this answers is not in the document]
Comment (josh-chamberlain) [reply setup_gui/README.md:1]: Probably not, I think we should leave it until we move the GUI

File: starter_template/README.md
[the remark this answers is not in the document]
Comment (josh-chamberlain) [reply starter_template/README.md:1]: Made it tinier so it's a true template/placeholder

File: USA/all_fields_extractor.py
[the remark this answers is not in the document]
Comment (josh-chamberlain) [reply USA/all_fields_extractor.py:1]: deleted!

**Review by thejqs — APPROVED** [review]
👍 🚢

**Pull request merged**